hack/update-bindata.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o pipefail
+
+if [ -z ${KUBE_ROOT} ]; then
+	echo "KUBE_ROOT not detected, setting default."
```

Comment:
can we maybe print out this message only if `KUBE_ROOT` *is* set, then? since the common case is that it isn't set, I'd rather reduce log spam.

also the conditional on L20 should use the [[ ]] conditional form. and probably use a default for KUBE_ROOT, so
```shell
if [[ -z "${KUBE_ROOT:-} ]]; then